.github/workflows/build_and_deploy.yml

```diff
@@ -0,0 +1,27 @@
+name: Build and Deploy Airmap Platform SDK
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with: 
+          submodules: recursive  
+      - name: Build Airmap Platform SDK for Android and QT 5.12.3 for x86 and armv7
+        run: docker build . --file docker/android-qt --tag airmap:qt
+      - name: Copy Release Binaries
+        run: |
+          docker create -ti --name dummy airmap:qt bash
+          docker cp dummy:/usr/airmapd/android/Qt.5.12/armv7/lib/libairmap-qt.so ./libairmap-qt-armv7.so
+          docker cp dummy:/usr/airmapd/android/Qt.5.12/x86/lib/libairmap-qt.so ./libairmap-qt-x86.so
```

Comment:
I would expect the generated SDK in a tar.gz, and have a structure like Qt.5.12/x86/libairmap-qt.so and Qt.5.12/armv7/libairmap-qt.so. There are rules in DataPilot looking for Airmap Platform SDK at compile time. Check https://github.com/YUNEEC/qgroundcontrol/pull/2034/commits/1524b27b6f363eca98d1962085e59f4d187c885d

Reply:
Good idea. It also would make sense to export the headers the same way and add it into one zip. 

Meanwhile, the libraries are copied to https://github.com/YUNEEC/airmap-platform-sdk/releases when doing a tagged GitHub release.

Reply:
Done :+1: